contracts/DCAStrategies/DCAStrategies/DCAStrategiesPermissionsHandler.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol';
+import '../../interfaces/IDCAStrategies.sol';
+
+abstract contract DCAStrategiesPermissionsHandler is IDCAStrategiesPermissionsHandler, ERC721Enumerable {
```

Comment:
Using `ERC721Enumerable` is very expensive in terms of gas costs. But we may need to have a 
`function totalSupply() external view returns (uint256)` exposed like [here](https://github.com/Mean-Finance/dca-v2-core/blob/main/contracts/interfaces/IDCAPermissionManager.sol#L7)

```suggestion
abstract contract DCAStrategiesPermissionsHandler is IDCAStrategiesPermissionsHandler, ERC721 {
```